platform/podjs/rollup-plugin-gen-podjs/genPodjs.js

```diff
@@ -41,4 +41,4 @@ function loadManifest(options = {}) {
     executeReplacement(podJsPath, options.manifestPath);
 }
 
-exports.default = loadManifest;
+module.exports = loadManifest;
```

Comment:
OK, I see that. Is genPodjs.js only used from these rollup configuration modules? If it's not, would it be affected? (That would probably be revealed by build scripts, but just in case)

Reply:
Would it be a problem is you leave *both*, just in case?

Reply:
All Rollup configuration scripts that use `genPodjs.js` have been converted to `.mjs`. Also `module.exports` should work as well when imported from a transpiled configuration script. Note that `@polypoly-eu/rollup-plugin-copy-watch` which is used from transpiled configuration scripts is using `module.exports` as well.

Reply:
As I said, we already import CommonJS modules that provide a default export via `module.exports` both from transpiled and native Rollup configuration scripts. `module.exports` works with pretty much every CommonJS implementation (including webpack), `exports.default` on the other hand is the odd one that only works in certain setups. Note that `@polypoly-eu/rollup-plugin-copy-watch` as well as various third-party modules we use already use `module.exports` only. So I don't see the point of keeping `exports.default` (in addition to `module.exports`) here.

Reply:
Well. we had two copies of `exports.default`. One here, and the other one above, in the other file modified in this PR: https://github.com/polypoly-eu/polyPod/pull/903/files#diff-981c1e9fce1052dc19957560ee3f7044619e7f7bfd4dd3758e15982d032cafb8L11
One has been substituted by `export default`, another by `module.exports`, so we no longer have coherence in the same kind of module, rollup plugins.
Fair enough, we might not want to keep the old one. But if we change them *both*, as you do in this PR, could you at least consider changing them *in the same way*? I'm good with either, your argument for using `module.exports` is probably good for `rollup-plugin.js`

Reply:
This PR was only about converting `silly-i18n/rollup-plugin` to ESM as a follow up to #860. Adjusting the export in `genPod.js` was merely necessary so that it can be imported by native Node.js. But fine, converted `genPod.js` to ESM as well now.